public/app/plugins/datasource/graphite/dashboards/metrictank.json

```diff
@@ -62,7 +51,7 @@
       "bars": false,
       "dashLength": 10,
       "dashes": false,
-      "datasource": "$datasource",
+      "datasource": "${DS_NAME}",
```

Comment:
this dashboard has a datasource variable (a templating variable) so this change is not correct as it will break the datasource selection at the top of the dashboard